plugins/chat/assets/stylesheets/common/base-common.scss

```diff
@@ -198,40 +198,9 @@ body.has-full-page-chat {
   grid-template-columns: var(--full-page-sidebar-width) 1fr;
   background: var(--d-content-background);
 
-  .chat-full-page-header {
-    border-bottom: 1px solid var(--primary-low);
-    background: var(--secondary);
-    z-index: 3;
-    display: flex;
-    align-items: center;
-
-    &__back-btn {
-      width: 40px;
-      min-width: 40px;
-      display: flex;
-      align-items: center;
-      justify-content: center;
-    }
-
-    .chat-channel-title {
-      .category-chat-name,
-      .chat-name,
-      .dm-usernames {
-        color: var(--primary);
-        display: inline;
-        text-overflow: ellipsis;
-        white-space: nowrap;
-      }
-    }
-
-    .-not-following {
-      .chat-channel-title {
-        max-width: calc(100% - 50px);
-      }
-      .join-channel-btn {
-        margin-left: auto;
-      }
-    }
+  .c-navbar-container {
+    position: sticky;
+    top: var(--header-offset);
```

Comment:
```suggestion
    top: var(--header-offset);
    height: var(--chat-header-offset);
    padding-inline: 1rem;
```

Reply:
It was to add a bit more space here, but it's fine, it's a nitpick
![image](https://github.com/discourse/discourse/assets/66427541/234f9d2c-7bbc-46fb-b134-cf5453423cc1)

Reply:
Yes I don't disagree... now that the navbar is the same everywhere, it will be easier for @chapoi to make some tweaks in css based on context